Makefile

```diff
@@ -113,3 +114,14 @@ gendocs: docs/minikube.md
 docs/minikube.md: $(shell find cmd) $(shell find pkg/minikube/constants) pkg/minikube/cluster/assets.go
 	$(MKGOPATH)
 	cd $(GOPATH)/src/$(REPOPATH) && go run -ldflags="$(K8S_VERSION_LDFLAGS) $(MINIKUBE_LDFLAGS)" -tags gendocs gen_help_text.go
+
+.PHONY: deb
+deb: 
+	@echo -e "Build .deb package in directory: out/.\n  Note:  There must compiled binary at out/minikube to put in the deb \n"
+	mv installers/linux/deb/minikube_* installers/linux/deb/minikube_$(DEB_VERSION) 2>/dev/null || true
```

Comment:
This is done to rename the dir as the folder should have the same name/versioning as the version of the package for deb in what I read (although it might not be necessary).  As such, moving this folder again to out/ will have the same # of steps as moving the .deb to out.  I could make the base folder have a version template (0.0-0) and then move it to out and do everything there, would that be better?  Please let me know if I misunderstood something, thanks.